schemas/common/page.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/page",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Page Object",
+  "type": "object",
+  "description": "Page object contains paging information for related resource when resource result is paginated.",
+  "definitions": {
+    "page": {
+      "properties": {
+        "orderBy": {
+          "title": "Order by property",
+          "type": "string",
+          "description": "The property by which this page is ordered.\n"
+        },
+        "order": {
+          "title": "Sort order",
+          "type": "string",
+          "pattern": "^(asc|desc)$",
+          "description": "Sort order, either \"asc\" or \"desc\".\n"
+        },
+        "start": {
+          "title": "First value",
+          "type": "string",
+          "description": "The first value, in sorted order, of the orderby property on this page.\n"
+        },
+        "next": {
+          "title": "Next page start value",
+          "type": "string",
+          "description": "The start value for the next page.\n"
+        },
+        "property": {
+          "title": "Filter properties",
+          "type": "string",
+          "description": "The list of properties by which the result is filtered, if any.\n"
+        },
+        "type": {
+          "title": "Filter type value",
+          "type": "string",
+          "description": "The applied type filtering value, if any.\n"
```

Comment:
How is `type` related to `filter`? Is this just a special, inconsistent way to filter for `dc:type` values?

Reply:
`type` filters on the media type of the list entry, while `property` filters on the existence of a specific property in a list entry.